zephyr/src/indigo_api_callback_dut.c

```diff
@@ -1500,45 +1490,24 @@ static int send_ap_disconnect_handler(struct packet_wrapper *req, struct packet_
     ret = snprintf(buffer, sizeof(buffer), "DISASSOCIATE %s reason=1", address);
     CHECK_RET(ret);
     /* Send command to hostapd UDS socket */
-    resp_len = sizeof(response) - 1;
-    wpa_ctrl_request(w, buffer, strlen(buffer), response, &resp_len, NULL);
-    /* Check response */
-    if (strncmp(response, WPA_CTRL_OK, strlen(WPA_CTRL_OK)) != 0) {
-        indigo_logger(LOG_LEVEL_ERROR, "Failed to execute the command. Response: %s", response);
-        message = TLV_VALUE_HOSTAPD_RESP_NOT_OK;
-        goto done;
-    }
+    RUN_QT_COMMAND(buffer, ret);
```

Comment:
you can also move the snprintf to `run_qt_command` then no need for CHECK_SNPRINTF